pyTigerGraph/gds/gsql/dataloaders/hgt_loader.gsql

```diff
@@ -72,20 +79,40 @@ CREATE QUERY hgt_loader_{QUERYSUFFIX}(
         # Get seeds
         IF input_vertices.size()==0 THEN
             start = {seed_types};
+            HeapAccum<ID_Tuple> (1, tmp_id ASC) @@batch_heap;
+            @@batch_heap.resize(batch_s);
```

Comment:
Is the resize() here due to that capacity in the heap definition cannot be a variable?

Reply:
Unfortunately that is the case.